aten/src/ATen/native/README.md

```diff
@@ -328,6 +328,15 @@ will be automatically differentiated! This can be the case if the function imple
 only calls other operations which are themselves differentiable.  In this
 case, you don't have to write an entry in `tools/autograd/derivatives.yaml`.
 
+### Will this function be exposed to python? What are the namespaces?
```

Comment:
Speaking of "will this function be exposed to python"... are these functions exposed to python? :)